src/Oscoin/Storage/Ledger.hs

```diff
@@ -0,0 +1,282 @@
+{-# LANGUAGE UndecidableInstances #-}
+
+-- | A 'Ledger' provides access to the blockchain as well as the state
+-- and receipt associated to a block in the blockchain.
+--
+-- This module should be imported qualified.
+module Oscoin.Storage.Ledger
+    ( Ledger
+    , hoist
+    , mkLedger
+    , newMemoryIO
+    , newMemoryFromGenesisIO
+    , newFromBlockStore
```

Comment:
Since this function also operates in `IO`, shouldn't it be consistently named with the others?